Makefile

```diff
@@ -83,6 +83,15 @@ clean-cli:  ## Clean CLI.
 clean-docker:
 	@(echo "DOCKER cleanup - NOT IMPLEMENTED YET " )
 
+test-lint:
```

Comment:
i would call it "go-lint" as it not running FE linter, and not to confuse it with "tests"

Reply:
oh we can run the ESLint as well. `test-` is the prefix for the rules that are related to testing. Linting is a test as well.